iothub/device/src/Serialization/SystemTestJsonPayloadConvention.cs

```diff
@@ -14,20 +14,20 @@ namespace Microsoft.Azure.Devices.Client
     /// </remarks>
     public class SystemTextJsonPayloadConvention : PayloadConvention
     {
-        internal static readonly Encoding s_encoding = Encoding.UTF8;
-
         private SystemTextJsonPayloadConvention() { }
 
         /// <summary>
         /// A static instance of this class.
         /// </summary>
-        public static readonly SystemTextJsonPayloadConvention Instance = new();
+        public static SystemTextJsonPayloadConvention Instance => new();
```

Comment:
interesting syntax

Reply:
This makes it a property, not a field. The readonly for properties is that there is no setter. In this case, as there is only a getter, there is a lamba implementation.

However, I realize this is wrong because this will yield a new object on every call. I'll update it to be `{ get; } = new();`